src/eia930.py

```diff
@@ -437,6 +441,13 @@ def manual_930_adjust(raw: pd.DataFrame):
     ovec_col = get_int_columns("PJM", raw.columns, ["OVEC"])
     raw.loc[:, ovec_col] = raw.loc[:, ovec_col] * -1
 
+    # Interchange AZPS - SRP is wonky. Use SRP - AZPS (inverted)
+    azps_srp = get_int_columns("AZPS", raw.columns, ["SRP"])
+    srp_azps = get_int_columns("SRP", raw.columns, ["AZPS"])
+    raw.loc[:, azps_srp] = (raw.loc[:, srp_azps] * (-1)).rename(
```

Comment:
We might want to set a date range for this so that it is only replaced pre 6/1/2020

Reply:
Also, do we need to also update the total interchange value?

Reply:
Good catch, total interchange does need to be changed. here's the raw data:
<img width="752" alt="Screen Shot 2022-09-06 at 10 42 17 PM" src="https://user-images.githubusercontent.com/12755256/188966040-87fbc42f-c943-4bfb-9af6-16d282c48094.png">

Reply:
Same series after fix. 

![Screen Shot 2022-09-07 at 4 18 06 PM](https://user-images.githubusercontent.com/12755256/188969414-47d4bbf6-652f-428d-93b9-47d7ca4c1b22.png)